getting_started.md

```diff
@@ -287,6 +287,43 @@ use the [analysis runner](https://github.com/populationgenomics/analysis-runner)
 which builds a Batch pipeline from a specific commit in a GitHub repository by
 running the driver itself on Hail Batch.
 
+### Analysis runner
+
+The analysis-runner builds a Batch pipeline from a speific commit in a GitHub repository. You can interact with an analysis server with the CLI. This can be installed with:
+
+```bash
+conda install -c cpg analysis-runner
+```
+
+Make sure you have the name of the dataset, as this corresponds to a specific analysis server and approved repository.
```

Comment:
We should probably expand this a bit and link to the storage policies doc. Before people can access a dataset (and use the runner for it), they need to get added to the corresponding permissions group. Maybe documenting the common failure case (no permissions) here might be helpful.

Reply:
I've changed this slightly to suggest that permissions are required, but not how you might attain those permissions.
Do you have a suggestion for how to frame this?